src/Model/MessageEnvelop.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Kfz24\QueueBundle\Model;
+
+use JMS\Serializer\Annotation as Serializer;
+
+/**
+ * Class MessageEnvelop
+ * @package Kfz24\QueueBundle\Model
+ */
+class MessageEnvelop
+{
+    /**
+     * @var MessageInterface
+     * @Serializer\MaxDepth(1)
+     */
+    protected $message;
+
+    /**
+     * @var \DateTimeImmutable
+     * @Serializer\Type("DateTimeImmutable")
+     */
+    protected $createdAt;
+
+    /**
+     * AbstractEnvelop constructor.
+     * @param MessageInterface $message
+     */
+    public function __construct(MessageInterface $message)
+    {
+        $this->message = $message;
+    }
+
+    /**
+     * @return int
+     * @Serializer\VirtualProperty("version")
+     */
+    public function getVersion(): int
+    {
+        return $this->message->getVersion();
+    }
+
+
+    /**
+     * @return string
+     * @Serializer\VirtualProperty("type")
+     */
+    public function getType(): string
+    {
+        return $this->message->getType();
+    }
+
+    /**
+     * @return MessageInterface
+     */
+    public function getMessage(): MessageInterface
+    {
+        return $this->message;
+    }
+
+    /**
+     * @return \DateTimeImmutable
+     */
+    public function getCreatedAt(): \DateTimeImmutable
+    {
+        return $this->createdAt;
+    }
+
+    /**
+     * @param \DateTimeImmutable $dateTime
+     * @return $this
+     */
+    public function setCreatedAt(\DateTimeImmutable $dateTime)
+    {
+        $this->createdAt = $dateTime;
+        return $this;
```

Comment:
Blank line missing.

Reply:
We need to enable those github checks in this repo, so we can have same quality standards as for search-service.